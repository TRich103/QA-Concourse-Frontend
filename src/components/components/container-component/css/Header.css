.navbar-light .navbar-nav .nav-link{
	color:black !important;
}
.navbar-light .navbar-nav .nav-link:hover{
	background-color:#d6d8da;
	color:Black;
}
.display_name{
	margin-top:8px;
	margin-right:5px; 
}
.dropdown-menu.show{
	width:200px !important;
	margin-left:-5%;
	margin-top:5%;
	padding:5px 5px!important;
	background-color:#d6d8da !important;
	font-size:16px;
	z-index:2;
}
.dropdown-item{
	padding:0px 0px!important;
}
.dropdown-item:hover{
	background: rgb(0, 64, 80,30%)!important;
	padding:0px 0px!important;
}
.menu{
    border:1px solid rgb(0, 64, 80,30%);    
    -moz-border-radius:50%;
    -webkit-border-radius:50%;
    width:100px;
	display:block;
	width:50px;
	height:50px;
	border-radius:50px;
	font-size:20px;
	line-height:100px;
	text-align:center;
	text-decoration:none;
	background-color:rgb(0, 64, 80,90%)}
.menu:hover{
		color:#ccc;
		text-decoration:none;
		opacity:0.9;
		}
.nav-item{
	color:black;
}
.sidebar_btn{
	cursor: grab;
}
#navigation-bar{
	width:100%;
}

.navbar,
.navbar-default {
    background-image: linear-gradient(to bottom,#f3f2f2 0,#d6d8da 100%) !important; /* override background image gradient w/ flat color */
}

#loginBtn{
  background-color: #004050;
	margin-top: 10px;
}

#loginBtn:hover{
  background-color: white;
  color:#004050;
  border: 1px solid #004050;
}

#logoutBtn{
  background-color: #2c3a3d;
}

#logoutBtn:hover{
  background-color: white;
  color:#004050;
  border: 1px solid #004050;
}
