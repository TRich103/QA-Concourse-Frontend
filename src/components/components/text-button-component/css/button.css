#Btn {
    background-color: #004050;
    border: 1px solid #004050;
}

#Btn:hover {
        background-color: white;
        color: #004050;
        border: 1px solid #004050;
}