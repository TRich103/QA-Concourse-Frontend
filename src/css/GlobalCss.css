.createTrainee{
    background-color: white;
    margin-top:5%;
    border-radius: 4px;
}

.createTraineeForm{
    height: 80%;
    width: 100%;
    margin-right: auto;
    margin-left: auto;
    padding-top: 5%;
    padding-bottom: 5%;
}

.DayPickerInput-Overlay {
    top: -332px;
}

.title{
    margin-left: 5%;
}

.text-input-fields {
    width: 55%;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
}

#createTraineeBtn {
	background-color: #005b72;
	width: 30%;
    margin-left: auto;
    margin-right: auto;
	text-transform: capitalize;
	font-family: "Roboto", sans-serif;
}

#createTraineeBtn:hover {
    background-color: white;
    color: #004050;
    border: 1px solid #004050;
}

.createUser{
    background-color: white;
    margin-top: 5%;
    border-radius: 4px;
}

.form-control {
    text-align: center;
    width: 10px;
    size: 5;
    color: blueviolet;
    background-color: red;
}

.addForm{
    height: 80%;
    width: 100%;
    margin-right: auto;
    margin-left: auto;
    padding-top: 2.5%;
    padding-bottom: 2.5%;
}

#updateBtn {
	background-color: #004050;
	width: 25%;
    margin-left: auto;
    margin-right: auto;
	text-transform: capitalize;
	font-family: "Roboto", sans-serif;
}

#dropDown{
    width:40%;
    margin-left: 30%;
    text-align: center;	
}


/* Change style of Role button and dropdown*/

button.dropdown-toggle.btn.btn-secondary {
    width: 60%;
    background-color: #005b72;
    margin-left: auto;
    margin-right: auto;
text-transform: capitalize;
font-family: "Roboto", sans-serif;
	font-size: 18px;
}

div.dropdown-menu.show {
    background-color: white;
    width: 60%;
    min-width: 0;
}
button.dropdown-item {
    color: black;
    text-align: center;
}
button.dropdown-item:hover {
    color: black;
    background-color: rgb(202, 202, 202);
    text-align: center;
}

/* Move title in line input elements page*/
h3 {
    width: 90%;
    margin-left: auto;
    margin-right: auto;
    text-align: center;
}

/* Shorten text input fields*/
.user-text-inputs {
    width: 40%;
    margin-left: auto;
    margin-right: auto;
}

.changePassword{
    background-color: white;
    margin-top:5%;
    border-radius: 4px;
    width:80%;
    margin-left:auto;
    margin-right: auto;
}

.changeForm{
    height: 80%;
    width: 80%;
    margin-right: auto;
    margin-left: auto;
    padding-top: 5%;
    padding-bottom: 5%;
}

.resetPassword{
    background-color: white;
    margin-top:5%;
    border-radius: 4px;
    width:80%;
    margin-left:auto;
    margin-right: auto;
}

.resetPassword p {
    text-align: center;
}

#updatePasswordBtn {
	background-color: #004050;
    width: 50%;
    border: 1px solid #004050;
}

#updatePasswordBtn:hover {
	background-color: white;
    color: #004050;
    border: 1px solid #004050;
}

.reportTitle{

}

.reportDetails{

}

.edit-form{
	background-color:white;
	width:80%;
	margin-left: auto;
	margin-right: auto;
	border-radius: 4px;
}


.form-group{
    text-align: center;
	width: 90%;
	margin-left: auto;
	margin-right: auto;
}

.all-edit-box{
	padding-top:5%;
	padding-bottom:5%;
	text-align: center;
}

#bursaryDates{
    height: 1%;
    width: 90%;
    margin-right: auto;
    margin-left: auto;
    padding-top: 1%;
    padding-bottom: 1%;
} 

#WorkingDays{
	height: 1%;
    width: 50%;
    margin-right: auto;
    margin-left: auto;
    padding-top: 1%;
    padding-bottom: 1%;
}

#updateBtn {
	font-family: "Roboto", sans-serif;
	font-size: 18px;
	border:1px solid white;
	background-color: #004050;
	color: white;
	border-radius: 0.25rem;
	opacity: 0.8;
}

#cancelBtn{
	width: 125px;
	height: 45px;
	position: relative;
	left: 40%;
	font-family: "Roboto", sans-serif;
	font-size: 22px;
	border:1px solid white;
	background-color: #004050;
	color: white;
	border-radius: 0.25rem;
	opacity: 0.8;
}

#cancelBtn:hover{
	background-color: white;
	color: #004050;
	border: 1px solid #004050;
}

.updateBtn:hover {
	opacity: 1;
	background-color: white;
	color: #457581;
}

#accountnumber {
	width: 100%;
}

#sortcode {
	width: 100%;
}

.validated-field-container {
	display: inline-flex;
	min-width: 40%;
	justify-content: center;
}

#accountImg {
	min-width: 48px;
	visibility: hidden;
}

#sortImg {
	min-width: 48px;
	visibility: hidden;
}

input[type=number]::-webkit-inner-spin-button, 
input[type=number]::-webkit-outer-spin-button { 
  -webkit-appearance: none; 
  margin: 0; 
}

#settings{
	margin-top: 5%;
}

.adminSettings{
	height:500px;
}

.settingTitle {
	padding: 2%;
}

.formSettings{
	width: 60%;
	margin-left: auto;
	margin-right: auto;
	padding: 2%;
}

#updateBtn:hover {
	background-color: white;
	color: #004050;
	border: 1px solid #004050;
}

#gSFrame{
	margin-top: 100px;
	border-radius: 5px;
}

.gSTitle{
	margin: auto;
	font-family: "Roboto", sans-serif;
	font-size: 40px;
	box-shadow: 5px 10px #004050;
	opacity: 0.8;
	border-radius: 5px;
	width: 50%;
}

#gS{
	height: 200px;
	color: white;
	background-color: #457581; 
	padding: 20px;
	border-left: solid 5px #457581;
	border-top: solid 5px #457581;
	border-right: solid 5px #457581;
	border-top-right-radius: 30px;
	border-top-left-radius: 30px;
}

.gSLabel{
	font-family: "Roboto", sans-serif;
	font-size: 20px;
}

#gSInput{
	font-family: "Roboto", sans-serif;
	font-size: 20px;
	width: 75px;
}

#gSUpdateBtn {
	width: 20%;
	font-family: "Roboto", sans-serif;
	font-size: 18px;
	border:1px solid white;
	background-color: #004050;
	color: white;
	border-radius: 0.25rem;
	opacity: 0.8;
}

#gSUpdateBtn:hover {
	background-color: white;
	color:  #004050;
}

.QAtable{
    margin: 0 auto;
    background-color: white;
    border-radius: 4px;
}

.QAtableMenu{
    margin: 0 auto;
    background-color: white;
    border-radius: 4px;
    width :100%;
    margin-left: 300px; 
}

.historyTable{
    margin: 0 auto;
    background-color: white;
    border-radius: 4px;
}

.BigBox{
    padding-top: 5%;
    width: 60%;
    margin-left: auto;
    margin-right: auto;   
}

h2{
    text-align:center;
}

.logTable{
    overflow: scroll;
}


.QAtable td{white-space:nowrap;}

#createExpenseBtn {
	background-color: #004050;
    width: 40%;
    border: 1px solid #004050;
}

#createExpenseBtn:hover {
    background-color: white;
    color: #004050;
    border: 1px solid #004050;
}

#OtherInput{
    margin-right: 28%;
    float: right;
    padding-top: 2%;
    padding-bottom: 2%;
}


#addUser{
    float:right;
    padding-right: 60px;
}

.link{
    color: white;
    text-decoration: none;
}
.link:hover {
    color:white;
    text-decoration: none;
}

.qabtn{
    background-color: #337ab7;
    border-radius: 4px;
    text-decoration: none;
    border:none;
}

.action{
    width:150px;
}

.heading{
    background-color:aliceblue;
}


#email a{
    text-decoration: none;
}

.trainee:hover{
	cursor: pointer;
}

.userRole{
	text-transform: capitalize;
}

#resultsTable{
    width: 100%;
    overflow :scroll;
}

#fakeBtn {
    background-color: #9ea5a594;
    border-color: #d5dbdb;
    color: #d5dbdb;
    cursor: default;
    opacity: 0.7;
    border:1px solid transparent;
    border-radius: 0.25rem;
    opacity: 0.8;
} 

#addUser{
    float:right;
    padding: 5px;
}

.link{
    color: white;
    text-decoration: none;
}

a.link:hover{
    color: #004050;;
}

#addUser .qabtn{
	width: 145px;
	height:30px;
    background-color: #004050;
    opacity: 0.8;
    border-radius: 4px;
    text-decoration: none;
    border:none;
    color: white;
    margin-right: 10px;
    border: 0.5px solid #004050;
	opacity: 0.8;
}

#addUser .qabtn:hover{
    background-color: white;
    border: 0.5px solid #004050;
    color: #457581;
    margin-right: 10px;
}

a:hover{
    color: #457581;
}

.modal-content{
    width: -70%;
}

.action{
    width:150px;
}

.bigBox{
    margin-top: 5%;
}

.filter{
    margin: 0.35%;
}
.filter-btn{
    border:1px solid transparent;
    background-color: #004050;
    opacity: 0.65;
    color: white;
    border-radius: 0.25rem;
    margin-left: 0.3%;
}
.filter-btn:hover{
    background-color: white;
    color : #004050;
    border-radius: 0.25rem;
    margin-left: 0.3%;
    border: 2px solid #004050;
}

.mod-body{
    text-align: center;
    margin-right: 6%;
}
.resetBtn{
    border:1px solid transparent;
    background-color: #004050;
    opacity: 0.65;
    color: white;
    border-radius: 0.25rem;
}

#email a{
    text-decoration: none;
}

.actionBtn{
    border:1px solid transparent;
    background-color: #004050;
    opacity: 0.65;
    color: white;
    border-radius: 0.25rem;
    opacity: 0.8;
}

.actionBtn:hover{
    background-color: white;
    color : #004050;
    border-radius: 0.25rem;
    border: 1px solid #004050;
}

.traineessmall{
    font-size: 12px;
}

.traineesmedium{
    font-size: 16px;
}

.traineeslarge{
    font-size: 20px;
}

.trainees:hover{
	cursor: pointer;
}

.forgotBtn{
  margin-top:3%;
  background-color:#FFFFFF;
  border: none;
  font-size:14px;
}
.forgotBtn:hover{
  /* background-color: #F0F8FF; */
  text-decoration: underline;
  
}

.btn:hover {
  background-color: #F0F8FF;
}

@media all {
    .Login {
      width: 500px;
      padding: 3% 0 0;
      margin: auto;    
    }
  
    .Login form {
      display: flex;
      flex: 1;
      flex-direction: column;
      max-width: 800px;
      position: relative;
      z-index: 1;
      background: #FFFFFF;
      min-width: 450px;
      margin: 0 auto 300px;
      padding: 45px;
      text-align: center;
      box-shadow: 0 0 20px 0 rgba(0, 0, 0, 0.2), 0 5px 5px 0 rgba(0, 0, 0, 0.24);
    }

    .uname{
        font-size: 20px;
        font-family: "Roboto", sans-serif;
    }

    .psw{
        font-size: 20px;
        font-family: "Roboto", sans-serif;
	margin-top: 10px;
    }

    .btn{
        font-family: "Roboto", sans-serif;
        text-transform: uppercase;
        outline: 0;
        width: 100%;
        border: 0px;
        padding: 15px;
        background-color: antiquewhite;
    
        font-size: 20px;
        -webkit-transition: all 0.3 ease;
        transition: all 0.3 ease;
        cursor: pointer;
    }
  }
  
  #top-nav-bar li {
    display: inline;
    color:black;
    text-decoration:none;
}
#top-nav-bar li :hover {
    color: #004050;
    text-decoration:none;
}

.sidebar_btn{
    background-color: #f8f9fb;
    display: inline;
    font-size: 20px;
    padding: 20px;
    margin-left: 10px;
    border-radius: 10px;
    color:black;
}
.sidebar_btn:a{
    color:white;

}
.sidebar_btn:hover{
    background-color: #d9d9d9;
    border-bottom: 3px solid #00ddb5;
}

.display_name{
	margin-top:8px;
	margin-right:5px; 
}

.nav-item{
	color:black;
}
.sidebar_btn{
	cursor: grab;
}
#navigation-bar{
	position:relative;
	width:100%;
}
.nav-link:hover {
	background-color:aliceblue;
	cursor: default;
}
/* Individual item */
.bm-item {
  display: inline-block;

  /* Our sidebar item styling */
  text-decoration: none;
  margin-bottom: 10px;
  color: #212529;
  transition: color 0.2s;
}

/* Change color on hover */
.bm-item:hover {
  color: #5a6268;
}

/* The rest copied directly from react-burger-menu docs */

/* Position and sizing of burger button */
.bm-burger-button {
  position: absolute;
  width: 36px;
  height: 30px;
  left: 100px;
  top: 15px;
}

/* Color/shape of burger icon bars */
.bm-burger-bars {
  background: #373a47;
}

/* Position and sizing of clickable cross button */
.bm-cross-button {
  height: 24px;
  width: 24px;
}

/* Color/shape of close button cross */
.bm-cross {
  background: #bdc3c7;
}

/* General sidebar styles */
.bm-menu {
  background: #f8f9fa;
  padding: 2.5em 1.5em 0;
  font-size: 1.15em;
}

/* Morph shape necessary with bubble or elastic */
.bm-morph-shape {
  fill: #373a47;
}

/* Wrapper for item list */
.bm-item-list {
  color: #212529;
}

/* Styling of overlay */
.bm-overlay {
  background: rgba(0, 0, 0, 0.3);
}

#loginBtn{
  background-color: #004050;
	margin-top: 10px;
}

#loginBtn:hover{
  background-color: white;
  color:#004050;
  border: 1px solid #004050;
}

#logoutBtn{
  background-color: #2c3a3d;
}

#logoutBtn:hover{
  background-color: white;
  color:#004050;
  border: 1px solid #004050;
}

.priv-notice-title {
    text-align: center;
    font-size: 150%;
  }
  
.priv-text{
  padding:20px;
}

  .modal-style {
    width: 90%;
    z-index: 1040;
    position: absolute;
    overflow-y: auto;
    overflow-x: hidden;
    transform: translate(-50%,-50%);
    border: 1px solid #e5e5e5;
    background-color: white;
    box-shadow: 0 5px 15px rgba(0,0,0,.5);
    padding: 20px
  }
  
  .priv-btn{
    position: relative;
    max-width: 100px;
    left: 50%;
    margin-left: -50px
  }

  .ReactTable .rt-tbody{
    overflow: overlay;
    }

#cRTitle{
	font-family: "Roboto", sans-serif;
	margin-left: 0px;
	border-bottom: solid 5px black;
}

.cRStatus{
	border-radius: 5px;
}

.cRButton{
	border:1px solid #457581;
	background-color: #457581;
	opacity: 0.65;
	color: white;
	border-radius: 0.25rem;
	opacity: 0.8;
	margin-left: 10px;
}

.cRButton:hover{
	opacity: 1;
	background-color: white;
	color: #457581;
}

#individual {
	margin-left: 1%;
}

.-striped.-highlight>div:nth-child(1)>div:nth-child(2)>div:nth-child(odd) {
	
	background-color: #eaf3ff;
}

.system_wrapper{
	margin: 0 auto;
    background-color: #f8f9fa;
    border-radius: 4px;
    width:80%;
	height:700px;
	margin-top: 5%;
}

.system_body{
	width:90%;
	height:20px;

}

.system_body_logs{
	overflow-y: auto;
	top:55%;
	height:630px;
	width:90%;
	white-space: pre-wrap;
	padding:1%;
	margin-left:5%;
	background-color:aliceBlue;
}

.system_wrapper button,textareabutton{
	margin-top:2%;
	float:right;
	margin-right:2%;
	background-color:#5a6268;
	color:white;
}


ul.nav.nav-tabs {
    border-bottom: 0px solid #dee2e6;
    padding-left: 10%;
}
.QASearchBar{
    padding-top:15px;
    padding-left:10px;
}

.QATabs{
    padding-top: 5%;
    padding-bottom: 5%;
    margin: 0 auto;
}

.QATabs .nav-link{
    background-color: aliceblue;
}

.nav{
    width:80%;
}


.table-hover>tbody>tr:nth-child(odd),
.table-hover>tbody>tr:nth-child(odd) {
    background-color: aliceblue;
}

.table-hover>tbody>tr:nth-child(odd):hover,
.table-hover>tbody>tr:nth-child(odd):hover {
    background-color: #c6e4ff;
}

.table-hover>tbody>tr:nth-child(even):hover,
.table-hover>tbody>tr:nth-child(even):hover {
    background-color: #c6e4ff;
}


.tabletext{
    font-size: 16px;
}
