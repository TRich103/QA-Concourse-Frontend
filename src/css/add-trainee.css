.createTrainee{
	width:80%;
	height:900px;
	margin-left:auto;
	margin-right:auto;
    background-color: white;
}

.createTraineeForm{
    height: 80%;
    width: 100%;
    padding-top: 5%;
    padding-bottom: 5%;
}

.DayPickerInput-Overlay {
    top: -332px;
}

.title{
    margin-left: 5%;
}
.grid-container{
  display: grid;
  grid-template-columns: auto auto auto;
  padding: 10px;
}
.gird-item{
  border: none;
  padding: 20px;
  font-size: 16px;
}
.grid-dates{
	border: none;
	position:center;
	float:left;
	margin-left:10%;
	padding: 20px;
}
.grid-wrapper{
	display:grid;
	  grid-template-columns: auto auto auto auto;
	  padding:10px;
}
.text-input-fields {
    width: 55%;
    margin-left: auto;
    margin-right: auto;
}

#createTraineeBtn {
	background-color: #005b72;
	width: 30%;
	text-transform: capitalize;
	font-family: "Roboto", sans-serif;
}

#createTraineeBtn:hover {
	color:#005b72;
    background-color: white;
}