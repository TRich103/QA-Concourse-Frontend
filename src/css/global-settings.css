#gSFrame{
	margin-top: 100px;
	border-radius: 5px;
}

.gSTitle{
	margin-right:auto;
	margin-left:auto;
	margin-top:2%;
	font-family: "Roboto", sans-serif;
	opacity: 0.8;
	border-radius: 5px;
	width: 50%;
}
.trainee-settings-wrapper{
	height:400px;
	width:100%;
}
#gS{
	height: 300px;
	color: black;
	padding: 20px;
	border-top-right-radius: 30px;
	border-top-left-radius: 30px;
}

.gSLabel{
	font-family: "Roboto", sans-serif;
	font-size: 20px;
}

#gSInput{
	font-family: "Roboto", sans-serif;
	font-size: 20px;
	width: 75px;
}

#gSUpdateBtn {
	width: 20%;
	font-family: "Roboto", sans-serif;
	font-size: 18px;
	border:1px solid white;
	background-color: #004050;
	color: white;
	border-radius: 0.25rem;
	opacity: 0.8;
}

#gSUpdateBtn:hover {
	background-color: white;
	color:  #004050;
}