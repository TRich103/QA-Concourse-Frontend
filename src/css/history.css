
.historyTable{
    margin: 0 auto;
    background-color: white;
    border-radius: 4px;
}

.ReactTable .tr-th, .ReactTable .rt-td {
    white-space: unset
}

.BigBox{
    padding-top: 5%;
    width: 60%;
    margin-left: auto;
    margin-right: auto;   
}

h2{
    text-align:center;
}

.logTable{
    overflow: scroll;
}


.QAtable td{white-space:nowrap;}

#createExpenseBtn {
	background-color: #004050;
    width: 40%;
    border: 1px solid #004050;
}

#createExpenseBtn:hover {
    background-color: white;
    color: #004050;
    border: 1px solid #004050;
}

#OtherInput{
    margin-right: 28%;
    float: right;
    padding-top: 2%;
    padding-bottom: 2%;
}