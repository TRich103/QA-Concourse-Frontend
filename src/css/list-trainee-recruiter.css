
.QAtable{
    background-color: white;
    width:100%;
}
#addUser{
    float:right;
    padding: 5px;
}

.link{
    color: white;
    text-decoration: none;
}

a.link:hover{
    color: #004050;;
}

#searchLogs {
    float: left;
}
.editable_table:hover{
	cursor:pointer;
}
.ReactTable{
	margin-left:0.5%;
	width:100%;
	height:760px;
    }
	
.ReactTable.-striped .rt-tr.-odd{
    background-color: rgb(0,64,80,10%);
}
.ReactTable.-highlight .rt-tbody :not(.-padRow).rt-tr:hover{
	    background: rgb(0, 64, 80,30%);
}

.ReactTable .rt-thead .rt-th.-sort-asc{
	box-shadow: inset 0 3px 0 0 rgba(0, 64, 80);
}

#addUser .qabtn{
	width: 145px;
	height:30px;
    background-color: #004050;
    opacity: 0.8;
    border-radius: 4px;
    text-decoration: none;
    border:none;
    color: white;
    margin-right: 10px;
    border: 0.5px solid #004050;
	opacity: 0.8;
}

#addUser .qabtn:hover{
    background-color: white;
    border: 0.5px solid #004050;
    color: #457581;
    margin-right: 10px;
}

a:hover{
    color: #457581;
}

.modal-content{
    width: -70%;
}

.action{
    width:150px;
}

.bigBox{

}

.filter{
    margin: 0.35%;
}
.filter-btn{
    border:1px solid transparent;
    background-color: #004050;
    opacity: 0.65;
    color: white;
    border-radius: 0.25rem;
    margin-left: 0.3%;
}
.filter-btn:hover{
    background-color: white;
    color : #004050;
    border-radius: 0.25rem;
    margin-left: 0.3%;
    border: 2px solid #004050;
}

.mod-body{
    text-align: center;
    margin-right: 6%;
}
.resetBtn{
    border:1px solid transparent;
    background-color: #004050;
    opacity: 0.65;
    color: white;
    border-radius: 0.25rem;
}

#email a{
    text-decoration: none;
}

.actionBtn{
    border:1px solid transparent;
    background-color: #004050;
    opacity: 0.65;
    color: white;
    border-radius: 0.25rem;
    opacity: 0.8;
}

.actionBtn:hover{
    background-color: white;
    color : #004050;
    border-radius: 0.25rem;
    border: 1px solid #004050;
}

.trainees:hover{
	cursor: pointer;
}