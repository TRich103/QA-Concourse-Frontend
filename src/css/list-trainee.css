
#addUser{
    float:right;
    padding-right: 60px;
}
.nav-link:hover{
	background-color:#e6e6e7;
	cursor:pointer;
}
.navbar-light .navbar-nav .nav-link:hover{
	background-image:linear-gradient(to bottom,#f3f2f2 0,#d6d8da 101%) !important;
	cursor:pointer;
	color:Black !important;
}

.filter_collapse{
	width:90%;
	height:48px;
	margin-top:1%;
	margin-left:-10%;
}
.link{
    color: white;
    text-decoration: none;
}
.link:hover {
    color:white;
    text-decoration: none;
}

.qabtn{
    background-color: #337ab7;
    border-radius: 4px;
    text-decoration: none;
    border:none;
}

.action{
    width:150px;
}

.heading{
    background-color:aliceblue;
}


#email a{
    text-decoration: none;
}

.trainee:hover{
	cursor: pointer;
}

.userRole{
	text-transform: capitalize;
}

#resultsTable{
    width: 100%;
	height:700px;
}

#fakeBtn {
    background-color: #9ea5a594;
    border-color: #d5dbdb;
    color: #d5dbdb;
    cursor: default;
    opacity: 0.7;
    border:1px solid transparent;
    border-radius: 0.25rem;
    opacity: 0.8;
} 