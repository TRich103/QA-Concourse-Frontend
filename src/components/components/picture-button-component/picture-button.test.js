import React from 'react';
import ReactDOM from 'react-dom';
import Enzyme,{shallow, mount, render, configure} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import axios from "axios";
import MockAdapter from 'axios-mock-adapter';
import renderer from 'react-test-renderer';
import PictureButton from './picture-button.component';
import Image from './images/qa-image.png';
import './css/image.css'

configure({ adapter: new Adapter() });

describe('TextButton', () => {


  it('image click should call function', () => {
    const string = 'picture-button';
    const clickFn = jest.fn();

      const component = mount(<PictureButton id={string} onClick={clickFn}  />);
      
      component
        .find('img#picture-button')
        .simulate('click');
      expect(clickFn).toHaveBeenCalled();
      component.unmount(); 
    });

    it('should match the inline CSS passed into the style', () => {
        const imageStyle = {
            maxHeight: 360,
            maxWidth: 360
        }
        
        const component = shallow(<PictureButton style={imageStyle}/>);
      
        expect(component).toMatchSnapshot();
      });

      it('should be the image passed', () => {
        
        const image = '../images/square.jpg';
        
        const component = shallow(<PictureButton image={image}/>);
      
        expect(component).toMatchSnapshot();
      });

      it('should possess the id that has been passed', () => {
        
        const imageID = 'img';
        
        const component = shallow(<PictureButton id={imageID}/>);
      
        expect(component).toMatchSnapshot();
      });

});