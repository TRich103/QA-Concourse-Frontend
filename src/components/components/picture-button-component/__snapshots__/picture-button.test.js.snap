// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextButton should be the image passed 1`] = `
<div>
  <Bootstrap(Image)
    src="../images/square.jpg"
    style={
      Object {
        "cursor": "pointer",
        "maxHeight": 128,
        "maxWidth": 128,
      }
    }
  />
</div>
`;

exports[`TextButton should match the inline CSS passed into the style 1`] = `
<div>
  <Bootstrap(Image)
    style={
      Object {
        "maxHeight": 360,
        "maxWidth": 360,
      }
    }
  />
</div>
`;

exports[`TextButton should possess the id that has been passed 1`] = `
<div>
  <Bootstrap(Image)
    id="img"
    style={
      Object {
        "cursor": "pointer",
        "maxHeight": 128,
        "maxWidth": 128,
      }
    }
  />
</div>
`;
